Updating Operators (#2016)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api-overview/index.md

| `ne` | not equal | If the attribute value does not match the operator value, there is a match. |
| `pr`       | present (has value)   | If the attribute has a non-empty value, or if it contains a non-empty node for complex attributes, there is a match.                                                                                                                                                           |
| `sw`       | starts with           | The entire operator value must be a substring of the attribute value, starting at the beginning of the attribute value. This criterion is satisfied if the two strings are identical.                                                                                         |

> **Note:** Some objects don't support all the listed operators.

> **Note:** All `Date` values use the ISO 8601 format `YYYY-MM-DDTHH:mm:ss.SSSZ`.

### Attribute operators

| Operator | Description | Behavior                                                         |
| -------- | ----------- | --------                                                         |
| `and`      | Logical AND | The filter is only a match if both expressions evaluate to true. |

> | `not` | Logical NOT | The filter is only a match if no expression evaluates to true. | 

Reviewer: Perhaps we can borrow from here instead: https://tools.ietf.org/html/rfc7644#section-3.4.2.2

> The filter is a match if the expression evaluates to false.

Author: Done!

> | `not` | Logical NOT | The filter is only a match if no expression evaluates to true. | 

Author: ```suggestion
| `not` | Logical NOT | The filter is a match if the expression evaluates to false. | 
```
